pom.xml

```diff
@@ -521,12 +521,6 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
-                <groupId>org.clojars.runa</groupId>
-                <artifactId>conjure</artifactId>
-                <version>${conjure.version}</version>
```

Comment:
The code no longer compiles because this dependency was removed.  dependencyManagement in the pom does not add in a dependency, it just tells maven if a project needs this dependency use this particular version.  Without this here the remaining dependency in storm-core does not know which version to use.